app/views/hyrax/base/_attribute_rows.html.erb

```diff
@@ -21,9 +21,4 @@
 <% OregonDigital::GenericMetadata::PROPERTIES.map(&:to_sym).each do |attr| %>
```

Comment:
Add a `.reject` to remove `oembed_url` from the resulting array since the rendering happens below.